rules-java/tests/src/test/java/org/jboss/windup/rules/apps/java/DiscoverAdditionalProjectDetailsTest.java

```diff
@@ -93,6 +94,9 @@ public void testLog4jNameFound() throws Exception
             boolean jarVersionFound = false;
             for (ArchiveModel archiveModel : archiveService.findAllByProperty(ArchiveModel.ARCHIVE_NAME, "log4j-1.2.6.jar"))
             {
+                if (archiveModel instanceof DuplicateArchiveModel)
+                    archiveModel = ((DuplicateArchiveModel)archiveModel).getOriginalArchive();
+
```

Comment:
Did we have this test before? I see that down below it looks at archiveModel.getProjectModel() of log4j. I thought this was being identified as a 3rd-party lib and not turned into a ProjectModel - ?
Ideally please add some inline comments to the tests too. This is getting complicated :)